src/Services/LoggingService.php

```diff
@@ -116,17 +117,26 @@ private function getPlentyMarketLoggerInstance(string $method, string $reference
   }
 
   /**
-   * Maps data from the he associative array that Wayfair provides to the PlentyMarkets logging API's inputs
+   * Maps data from the associative array that Wayfair provides to the PlentyMarkets logging API's inputs
    *
    * @param $loggingInfo
    *
    * @return array
    */
   public function extractVars($loggingInfo): array {
+    $truncatedMsg =[];
+
+
     $additionalInfo = $loggingInfo['additionalInfo'] ?? [];
     $method = $loggingInfo['method'] ?? null;
     $referenceType = $loggingInfo['referenceType'] ?? null;
     $referenceValue = (int) $loggingInfo['referenceValue'] ?? null;
+
+    if (strlen(json_encode($loggingInfo['additionalInfo'])) > self::STRING_LIMIT) {
+      $truncatedMsg['message'] =  $loggingInfo['additionalInfo']['message'];
```

Comment:
who/what is setting `message` in `additionalInfo` ?

Reply:
`message` is by whatever function that call the the loggingService, I used `message` because all logs in PlentyMarket has a message property

Reply:
I think the `message` you're referring to is manually set by the developer and does not appear in all logs.

Reply:
Ok, I make the adjustments